enhancements/sig-policy/59-ansible-everyevent-mode/README.md

```diff
@@ -0,0 +1,209 @@
+using the `once` mode. If there were more noncompliant clusters at the time, then `target_clusters` would also contain
+those as well. The controller would then set the following status on the `PolicyAutomation` object. Note that there is
+currently no status that ever gets set on `PolicyAutomation` objects.
+
+```yaml
+status:
+  clustersWithEvent:
+    cluster1:
+      automationStartTime: "2022-06-22T23:59:30Z"
+```
+
+This status update would tell the Governance Policy Propagator controller to not run the Ansible job again on `cluster1`
+the next time it notices that the `enable-etcd-encryption` policy is noncompliant on `cluster1` such as during periodic
+reconciling or status updates. This isn't meant to store a complete history of which clusters had an Ansible job run due
+to a policy violation. It is meant to track the clusters **currently** in the state based on the `eventHook` value in
+the `PolicyAutomation` object and the appropriate automation was run for those clusters. Such a history can be found on
+the Ansible Tower/Ansible Automation Platform job history.
+
+Several minutes later, the `enable-etcd-encryption` policy becomes noncompliant on `cluster2`. The Governance Policy
+Propagator controller would run the Ansible job from the Hub while passing the `extra_var` of
+`target_clusters: ["cluster2"]`. It would then set the following status on the `PolicyAutomation` object.
+
+```yaml
+status:
+  clustersWithEvent:
+    cluster1:
+      automationStartTime: "2022-06-22T23:58:30Z"
+    cluster2:
+      automationStartTime: "2022-06-22T23:59:30Z"
+```
+
+Several minutes later, the `enable-etcd-encryption` policy becomes compliant on `cluster1`. The Governance Policy
+Propagator controller would not run an Ansible job and set the following status on the `PolicyAutomation` object.
+
+```yaml
+status:
+  clustersWithEvent:
+    cluster2:
+      automationStartTime: "2022-06-22T23:59:30Z"
+```
+
+Now consider the case where the user made a change to their Ansible job and wants the Governance Policy Propagator
+controller to rerun the Ansible jobs for all current noncompliant managed clusters. The user would set the existing
+annotation of `policy.open-cluster-management.io/rerun` to `true` on the `PolicyAutomation` object and thus signal the
+Governance Policy Propagator controller to ignore `status.clustersWithEvent` and run an Ansible job for every
+noncompliant managed cluster.
+
+### Limiting the Number of Ansible Jobs
+
+If a user has a policy that they are concerned with often going between compliant and non-compliant, they may not want
+to run an Ansible job every time the cluster becomes noncompliant in order to not overwhelm Ansible Tower/AAP. In this
+case, a new optional spec field of `delayAfterRunSeconds` (follows the
+[API naming conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions))
```

Comment:
`repeatEventDelaySeconds` could be a consideration? I don't know if it's any better in general, but it might clarify the behavior I commented on separately. Eg, this would be more clear that the triggering event `compliant->noncompliant` has to occur after this delay - ***if*** that's even the behavior we want, of course.

Reply:
I clarified the behavior in the doc. Could you please take another look?

Reply:
> To me it sounds like you are queueing up to 1 action during the delay time

That's right. Let me try to reword this to make it clearer.

Reply:
Thanks, yes.